test/nginx_aaad_proxy_test.py

```diff
@@ -154,6 +154,47 @@ def main(args):
         test_proxy_user_valid_permissions_read(app_url, headers)
         test_proxy_user_valid_permissions_delete(app_url, headers, json_data)
 
+    chronos_common_access_url = ['http://chronos.dev.roger.dal.moz.com/',
```

Comment:
Made the changes. The roger specific details ( which env the test will be executed ) will be provided by user as a command line argument.